pod/perlop.pod

```diff
@@ -2467,6 +2467,8 @@ list of lines (however you've defined lines with C<$/> or
 C<$INPUT_RECORD_SEPARATOR>), or an empty list if the shell (or command)
 could not be started.
 
+    print qx/date/; # prints "Sun Jan 28 06:16:19 CST 2024"
```

Comment:
This section of `perlop` documents both the `qx` operator and its backticks variant.  If we're going to add an example of the first, then perhaps we should also add one of the second:
```
    print `date`; # also invokes the 'date' command
```

Reply:
Yes, be my guest, the more examples the merrier! Thanks.